core/runtime/ops.rs

```diff
@@ -631,4 +637,100 @@ mod tests {
   // We don't actually test this one -- we just want it to compile
   #[op2(core, fast)]
   pub fn op_test_generics<T: Clone>() {}
+
+  /// Tests v8 types without a handle scope
+  #[allow(clippy::needless_lifetimes)]
```

Comment:
Can this be handled by the op itself so we don't need to spam these ignores everywhere in our code?

Reply:
This is only used here in the tests because the lifetime is useless and I wanted some tests that included it.